src/main/java/de/hysky/skyblocker/skyblock/tabhud/widget/hud/HudPowderWidget.java

```diff
@@ -1,17 +1,46 @@
 package de.hysky.skyblocker.skyblock.tabhud.widget.hud;
 
+import de.hysky.skyblocker.skyblock.dwarven.DwarvenHud;
 import de.hysky.skyblocker.skyblock.tabhud.util.Ico;
 import de.hysky.skyblocker.skyblock.tabhud.widget.Widget;
 import net.minecraft.text.MutableText;
 import net.minecraft.text.Text;
 import net.minecraft.util.Formatting;
 
+import java.text.NumberFormat;
+import java.text.ParseException;
+
 // this widget shows the status of the king's commissions.
 // (dwarven mines and crystal hollows)
 // USE ONLY WITH THE DWARVEN HUD!
 
 public class HudPowderWidget extends Widget {
 
+    /**
+     * American number format
+     */
+    private static final NumberFormat NUMBER_FORMAT = NumberFormat.getInstance();
```

Comment:
Remove the American note as the `NumberFormat` instance is based off the user's locale.